src/discord_bot/bot.py

```diff
@@ -40,7 +40,8 @@ async def ask_model(ctx: interactions.SlashContext, model: str = "", prompt: str
   await ctx.defer()
 
   response = await answer_question(model, prompt, AI_SERVER_URL)
-  await ctx.send(response)
+  for r in response:
+    await ctx.send(r)
```

Comment:
I don't think this should be the solution. It's better to limit this answer into 2000 chars and use `ctx.reply` instead of `ctx.send`.

Reason being: By using `ctx.send`, we have no idea what answer is for which prompt, esp. when a few people asks the model at the same time.